nodeList.json

```
@@ -1,13 +1,16 @@
 {
   "nodeList": [
     {
+      "routerModel": "edgerouter-x-sfp",
```

```suggestion
      "model": "er-x-sfp",
```

Collapse  models into one field
ex-x-sfp is the model number on the box